st2common/st2common/runners/base.py

```diff
@@ -288,7 +288,8 @@ def pre_run(self):
                                                   entry_point=self.entry_point,
                                                   worktree_path=self.git_worktree_path)
 
-            assert(entry_point.startswith(self.git_worktree_path))
+            if not (entry_point.startswith(self.git_worktree_path)):
+                raise ValueError("The entry point value does not match with git worktree path.")
```

Comment:
Please include the entry point and the git worktree path so the user has some idea of what to do to correct the issue.